src/solvers/dg.jl

```diff
@@ -142,6 +142,43 @@ function Base.show(io::IO, ::MIME"text/plain", integral::VolumeIntegralPureLGLFi
 end
 
 
+"""
+    VolumeIntegralStaggeredGrid
+
+A shock-capturing volume integral type for DG methods based on a subcell blending approach
+with a low-order FV method.
+
+!!! warning "Experimental implementation"
+    This is an experimental feature and may change in future releases.
+"""
```

Comment:
Maybe add a reference to the HG-type shock capturing approach? E.g.,

```suggestion

See also: [`VolumeIntegralShockCapturingHG`](@ref)
"""
```
?